### ISSUE TITLE
How many epoch is the model trainned that can reach the same accuracy(AEE) and outlier as the paper release?

### ISSUE THREAD

**Issue by congliu19** [Cyril-Sterling/EVFlowNet-pytorch#2]
I have trained the model for 100 epoch. But the result on sequence of outdoor_day1 is AEE 1.2 and Outlier 1.3.

**Comment by CyrilSterling** [comment]
Yeah, I think something get to be wrong. The code Alex gives in tensorflow has the same result with this. It works badly in the outdoorday1 while no problem in other datasets. In outdoorday1, the result can't reach the AEE and outlier as the paper release.

**Comment by congliu19** [comment]
Thanks for your reply. I want to make sure that do you mean both tensorflow version model by Alex and pytorch version model by you are all can't reach the same results as the paper release on outdoorday1?

**Comment by CyrilSterling** [comment]
Yes.

**Comment by congliu19** [comment]
Thanks a lot！
